predicators/spot_utils/graph_nav_maps/floor8-cup-table/metadata.yaml

```diff
@@ -38,7 +38,7 @@ known-immovable-objects:
     x: 0.0
     y: 0.0
     z: -0.5
-  white-table:
+  smooth_table:
```

Comment:
is this because you added the smooth table to put the ball on initially?

Reply:
yes!